tests/phpunit/tests/blocks/renderCommentTemplate.php

```diff
@@ -149,7 +149,7 @@ function test_build_comment_query_vars_from_block_sets_cpage_var() {
 
 		// This could be any number, we set a fixed one instead of a random for better performance.
 		$comment_query_max_num_pages = 5;
-		// We subtract 1 because we created 1 comment at the beginning.
+		// We substract 1 because we created 1 comment at the beginning.
```

Comment:
`subtract` was fine 😅 

```suggestion
		// We subtract 1 because we created 1 comment at the beginning.
```

Reply:
OMG, so sorry about that. 🙈 😅 It was fixed already and I typed it wrong again. 😇

Reply:
It is _substraer_ or _sustraer_ in Spanish, hence my confusion. ☝️